tests/eth-tx-auth.test.ts

```diff
@@ -473,7 +473,7 @@ describe('Ethereum Transaction Authenticator', function () {
       );
       expect.fail('Should have failed');
     } catch (err: any) {
-      expect(err.message).to.contain(shortString.encodeShortString('Invalid sender address'));
+      expect(err.message).to.contain(shortString.encodeShortString('Commit not found'));
     }
```

Comment:
could we add a test that tests the "attack" vector that OZ mentioned? Just to add it as a regression test for later work

Reply:
yeah sorry added but forgot to push